src/Common/Command/ImportLocaleCommand.php

```diff
@@ -2,117 +2,11 @@
 
 namespace Common\Command;
 
-use Exception;
-use Symfony\Component\Console\Command\Command;
-use Symfony\Component\Console\Input\InputInterface;
-use Symfony\Component\Console\Input\InputOption;
-use Symfony\Component\Console\Output\OutputInterface;
-use Backend\Modules\Locale\Engine\Model as BackendLocaleModel;
-
 /**
  * This is a simple command to install a locale file
+ *
+ * @deprecated moved to the Console\Locale namespace
  */
-class ImportLocaleCommand extends Command
+class ImportLocaleCommand extends \Console\Locale\ImportLocaleCommand
```

Comment:
I though it was a bit of overkill since the name of the class hasn't changed so I would need to add an alias for it.

Also this is just here for BC but normally nobody would end up here